### ISSUE TITLE
Selecting multiple objects displays wrong info

### ISSUE THREAD

**Issue by jes-r** [IfcOpenShell/IfcOpenShell#5837]
Hi,

I have 2 issues when selecting multiple objects:

1. Select an object, e.g. a window. Then decide you want to only select multiple walls to change a property. LMB + drag select the walls, but what happens? In object information etc. you still see the properties of the window. You have to shift + LMB a wall in the selection to make that the active object and show the properties of that wall. 

Of course this is wrong as you can edit the selections properties while you see the properties of the window. I believe this can also cause bugs.
![image](https://github.com/user-attachments/assets/2d7372a5-f9e3-422b-85f8-e762bab6ccb0)
![image](https://github.com/user-attachments/assets/3023a8af-7dd4-4f20-aa56-4bf41cb6c420)

2. Closely related to 1: If you have instances with different types selected (like walls but different typed walls), it shows the properties of only the active object (which doesn't need to be in the selection, see 1.)  If one wall has an occurence property and the other doesn't, you have no way of knowing.

Maybe the Revit way (showing <varies> or making a property uneditable) is a solution? Or another way, I don't really know.

In conclusion:

You should only see the properties of elements in a selection. It should be visible if elements have different properties.

**Comment by Andrej730** [comment]
That's just Blender UX - active object doesn't have to be part of the selection and UI is displaying information only for the active object.

> Of course this is wrong as you can edit the selections properties while you see the properties of the window

When you edit properties and save them it's changing properties only for the active object (unless pset is shared between multiple objects), not for all selected objects.

> it shows the properties of only the active object (which doesn't need to be in the selection, see 1.) If one wall has an occurence property and the other doesn't, you have no way of knowing.

If you want to perform batch changes and need to be aware of all the differences between properties of different selected objects, then you should probably use spreadsheet import/export.

**Comment by Moult** [comment]
Agreed, this is not a bug, this is inherently how Blender works.

**Comment by jes-r** [comment]
> That's just Blender UX - active object doesn't have to be part of the selection and UI is displaying information only for the active object.

Okay, so no change there I guess? is this preferred for the Bonsai user category too? 


> If you want to perform batch changes and need to be aware of all the differences between properties of different selected objects, then you should probably use spreadsheet import/export.

Ah okay, haven't looked into that. Seems promising!
